### ISSUE TITLE
Violations not reported for ES6 classes and methods

### ISSUE THREAD

**Issue by pyoor** [gajus/eslint-plugin-jsdoc#704]
## Expected behavior
Violations are reported for ES6 classes and methods.

## Actual behavior
No violations are reported.  Using the following example, I would expect `jsdoc/require-jsdoc` to be raised.

## ESLint Config
```js
module.exports = {
  env: {
    browser: true,
    node: true
  },
  root: true,
  parser: '@typescript-eslint/parser',
  plugins: [
    '@typescript-eslint',
    'jsdoc',
  ],
  extends: [
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:jsdoc/recommended'
  ],
}
```

## ESLint sample
```js
export class hello {
    world(): void {
        console.log("hello world");
    }
}
```

## Environment

- Node version: v14.16.0
- ESLint version: v7.22.0
- `eslint-plugin-jsdoc` version: v4.18.0

**Comment by brettz9** [comment]
Please see the `require` option, specifically the need to enable `ClassDeclaration: true` and `MethodDefinition: true` (and potentially `ClassExpression :true`). See also the `contexts` option for more advanced control.
